.eslintrc.js

```diff
@@ -1,26 +1,38 @@
-const path = require('path');
+const path = require("path");
 
 module.exports = {
   root: true,
-  extends: ['vinta/recommended'],
+  extends: ["vinta/recommended-typescript"],
   rules: {
-    "default-param-last": "off",  // due to initialState in Redux
-    "@babel/camelcase": "off"
+    "default-param-last": "off", // due to initialState in Redux
+    "import/extensions": [
+      "error",
+      "ignorePackages",
+      {
+        ts: "never",
+        tsx: "never",
+      },
```

Comment:
Why this one needs this never?

Reply:
This one is needed so we don't have to add `.ts/.tsx` when importing (e.g.: `import App from "./App.tsx"`). 

I am not sure why we are facing that, maybe some side effect of `eslint-config-vinta`, but I'm still checking...

Reply:
I spent some time on this, and it seems most of the solutions use the `import/extensions` with `never` options indeed.

Reply:
sounds good